### ISSUE TITLE
Missing alt/title options of inline image

### ISSUE THREAD

**Issue by strarsis** [WordPress/gutenberg#14781]
**Describe the bug**
The inline image is missing options for alt and title attributes.

**To Reproduce**
1. Open Gutenberg editor.
2. Insert inline image.
3. When the inline image is selected, there is only an option for the width shown.

**Comment by draganescu** [comment]
Still the case in Gutenberg 7.9.1

<img width="638" alt="Screenshot 2020-04-22 at 11 48 57" src="https://user-images.githubusercontent.com/107534/79961366-49b4d900-848f-11ea-805d-fb0376ff1830.png">

**Comment by skorasaurus** [comment]
When I try to reproduce this according to the original instructions (inserting a new inline image), the media modal pops up and I am prompted to enter or edit the alt text. 

With that said, the alt-text is not editable when you edit an existing post that contains an inline image. 
(@afercia mentioned this at https://github.com/WordPress/gutenberg/issues/10595#issuecomment-430900146) 

Do we want to keep this issue open and @strarsis could you edit your to description to reflect this update or close this issue and make a new one to reflect the issue?

**Comment by XVII** [comment]
The `title` attribute is missing entirely. Only the `alt` gets inserted when using an inline image assuming it was set during insert (can't be edited after).

Image blocks can have the `Alt` and `Title` set both before and after insert. They should be included alongside `width` as mentioned by OP.

![image](https://github.com/WordPress/gutenberg/assets/446329/05ff24b2-a983-48e7-8bc6-71dfa6ac47d0)

**Comment by Mamaduka** [comment]
The `alt` text field was added in #64124.  The `title` field is considered an advanced field for the Image, and considering the help note, I'm not entirely sure how useful/valid it is these days.

cc @joedolson, @t-hamano.

**Comment by t-hamano** [comment]
There is an effort to remove the `title` attribute from the core codebase: https://core.trac.wordpress.org/ticket/24766

Given that, I don't think the `title` attribute editing UI is necessary for inline images.

**Comment by t-hamano** [comment]
For now, let's close this issue for now.

@strarsis If I've missed something, please feel free to comment.

**Comment by joedolson** [comment]
I don't think there's any benefit to providing editing of the title attribute. It serves little to no purpose.

**Comment by Mamaduka** [comment]
Thanks for the input, @joedolson and @t-hamano!
